rclpy/test/test_executor.py

```diff
@@ -230,6 +230,12 @@ def __await__(self):
         rclpy.spin_once(self.node, timeout_sec=0)
         self.assertTrue(did_return)
 
+    def test_executor_add_node(self):
+        self.assertIsNotNone(self.node.handle)
```

Comment:
I see that this matches the style in this file but in general we should use a plain assert instead (the recommended way with `pytest`). Nothing needs to happen for this patch though...

Reply:
Roger. I'll make sure to use a plain `assert` in the future.